tools/browser_test_generator.js

```diff
@@ -33,13 +33,20 @@ module.exports = function(tests, options) {
     return Promise.join(promiseExport, main, function(promiseExport, main) {
         var browserify = require("browserify");
         var contents = promiseExport + "\n" + main + "\n" + testRequires;
-        var b = browserify({
+        var complete = browserify({
             basedir: baseDir,
             entries: stringToStream(contents)
         });
-        return Promise.promisify(b.bundle, b)().then(function(src) {
-            return writeFile(path.join(baseDir, "bundle.js"), src);
+        var worker = browserify({
+            basedir: baseDir,
+            entries: stringToStream(promiseExport),
         });
+        return Promise.join(
+            Promise.promisify(complete.bundle, complete)().then(function(src) {
```

Comment:
Nevermind, Petka mentioned that it needs to run in 0.10.x - guess we're in 2014 :D